build-system/tasks/helpers.js

```diff
@@ -522,6 +522,65 @@ async function compileUnminifiedJs(srcDir, srcFilename, destDir, options) {
   }
 }
 
+/**
+ * Transforms a given JavaScript file entry point with esbuild and babel, and
+ * watches it for changes (if required).
+ * Used by 3p iframe vendors.
+ * @param {string} srcDir
+ * @param {string} srcFilename
+ * @param {string} destDir
+ * @param {?Object} options
+ * @return {!Promise}
+ */
+async function compileJsWithEsbuild(srcDir, srcFilename, destDir, options) {
+  const startTime = Date.now();
+  const entryPoint = path.join(srcDir, srcFilename);
+  const destFilename = maybeToEsmName(
+    options.minify ? options.minifiedName : options.toName
+  );
+  const destFile = path.join(destDir, destFilename);
+
+  if (watchedTargets.has(entryPoint)) {
+    return watchedTargets.get(entryPoint).rebuild();
+  }
+
+  const plugin = getEsbuildBabelPlugin('unminified', /* enableCache */ true);
```

Comment:
I think we'll need to make a new babel config for `minified` callers as we have transforms we want performed in dist builds that we don't do for unminified builds (like stripping devAssert).

It may be as simple as copy/pasting the pre-closure transforms.

Reply:
@samouri Thanks for the suggestion! Updated the PR to include minified-config w/ subset of pre-closure transformers.